### DISCUSSION TITLE
Repo-based Media Issue

### DISCUSSION THREAD

**Discussion by ikiselev1989** [tinacms/tinacms#4232] [Help] [answered]
Trying to integrate tina in astro project. 
Not working media store on production. I made setup by [guide](https://tina.io/docs/frameworks/astro/)
![image](https://github.com/tinacms/tinacms/assets/22061871/dc6b23cd-665b-416d-a8a3-ee8cf4b8361b)
![image](https://github.com/tinacms/tinacms/assets/22061871/92f79532-06b5-4143-b19c-40c93e2bcd83)

If I move to MediaManager in admin I see list of images for a second but then I see this.
![image](https://github.com/tinacms/tinacms/assets/22061871/9af1b292-7aea-40b7-a87a-30e2bbd2686e)

**Comment by kldavis4** [comment]
what branch is tina currently setup on?

**Reply by ikiselev1989** [reply]
I have one branch - master, so it's setup on

**Reply by kldavis4** [reply]
what's your client id? (it is safe to share here)

**Reply by ikiselev1989** [reply]
58af7444-56d7-4ee4-90a5-cbc376a94a94

**Reply by kldavis4** [reply, marked as answer]
ok - try configuring the media branch again now in tina cloud

**Reply by ikiselev1989** [reply]
thanks. it's working now. So what it was? on my side?

**Reply by kldavis4** [reply]
For some reason the default branch for the repository was incorrectly set to `main` in Tina Cloud instead of `master`. Not sure if it is an issue in Tina Cloud or not, but it impacts your ability to configure the media branch.

**Reply by ikiselev1989** [reply]
Hi! I just made another one project for test an I faced the same issue. ClientId - `38aacc92-2d9f-4529-9ba8-feed29bd987d`.
Can you check this one please?

**Reply by kldavis4** [reply]
Can you provide details on what you did so we can get the issue fixed?

**Reply by ikiselev1989** [reply]
If I use "main" branch it's working right. If I forgot to rename "master" to "main" and push it to repo. It's not working after I change tina's config branch name to "master"

**Reply by kldavis4** [reply]
I tried to replicate this by taking a repository setting the default branch to foobar which already had tina installed. I created the project in tina cloud with that repository and was able to successfully configure media, so I think I'm missing a detail.

**Reply by ikiselev1989** [reply]
I got an error that "main" branch is not exist, than I go to config, change "main" to "master", push it. And got the issue with media.

**Reply by kldavis4** [reply]
can you list all of the steps (start to finish) to reproduce this? Or record a https://loom.com/ showing how to reproduce?
